vllm/model_executor/models/mixtral.py

```diff
@@ -465,6 +513,13 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
         ]
 
         expert_params_mapping = [
+            # These are the activation scales for the experts
```

Comment:
activation scales -> weight scales

Also, it would be more natural to put this after the weights I think (since the weights are always used, but the scales are only used for FP8) :)

Reply:
`ws_scale` must come before `ws` b/c of the logic below

Reply:
```python
for param_name, weight_name, expert_id in expert_params_mapping:
    if weight_name not in name:
        continue
```

Reply:
`w2s` is in `w2s_scale`, so we will not "continue" properly

Reply:
Ah, I see. That seems a little subtle and could easily lead to some debugging nightmares later -- how about we rename `ws` to `ws_weight` and `w2s` to `w2s` `w2s_weight`?

Reply:
Yeah thats a good approach - it certainly confused me at first + I should have left a comment

Reply:
Btw, it might also be a good idea to rename `ws` to `w13` and `w2s` to `w2`, feel free to do that if you want. That way it is much clearer how these correspond to the matrices in the checkpoint :)

Reply:
I like this renaming as well, made all the changes!